### PR TITLE
Change Statistics dir, add comments, remove unnecessary items from _c…

### PR DESCRIPTION
**Pull request by ignotus666**
…onfig.yml

# Does this need translation?

- [ ] Yes 
- [* ] No 

# Changes

Just some corrections and added a comment to po4a-cache script.

### CODE REVIEW

File: .github/workflows/add-lang.yml [hunk 1]
```diff
@@ -58,4 +58,4 @@ jobs:
         run: |
           docker run \
           -v ${{ github.workspace }}:/srv/jekyll -v ${{ github.workspace }}/_site:/srv/jekyll/_site \
-          jekyll/builder:latest /bin/bash -c "chmod a+w /srv/jekyll/Gemfile.lock && chmod 755 /srv/jekyll && jekyll build --future"
+          jekyll/builder:latest /bin/bash -c "chmod a+w /srv/jekyll/Gemfile.lock && chmod 777 /srv/jekyll && jekyll build --future"
```
Comment (ann0see) [inline .github/workflows/add-lang.yml:61]: Already changed that on this repo.

File: _config.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ plugins:
   - jekyll-minifier
 languages:                   ["en", "es", "de", "it", "fr", "pt"]
 default_lang:                "en"
-exclude_from_localization:   ["assets", "images", "css", "README.md", "manifest.json", "humans.txt", "sitemap.xml", "robots.txt", "/translator-files", "/po4a-*.sh", "404.html", "CNAME", "LICENSE.md"]
+exclude_from_localization:   ["assets", "images", "css", "README.md", "manifest.json", "humans.txt", "sitemap.xml", "robots.txt", "404.html", "CNAME", "LICENSE.md"]
```
Comment (ann0see) [inline _config.yml:7]: Thanks.

**Review by ann0see — COMMENTED** [review]
Thanks! Not sure about the change I already did.

File: .github/workflows/add-lang.yml (hunk 1 above)
Comment (ignotus666) [reply .github/workflows/add-lang.yml:61]: You changed it in the jekyll file but not in add-lang

Comment (ann0see) [reply .github/workflows/add-lang.yml:61]: Ok. Thanks

**Review by ann0see — APPROVED** [review]

**Pull request merged**